pkgs/applications/misc/khal/default.nix

```diff
@@ -46,6 +46,10 @@ in with python.pkgs; buildPythonApplication rec {
   nativeBuildInputs = [ setuptools_scm pkgs.glibcLocales ];
   checkInputs = [ pytest ];
 
+  postInstall = ''
+    install -D misc/__khal $out/share/zsh/site-functions/__khal
```

Comment:
Is `__khal` the same as `_khal`?

Reply:
I think so, at least it gets picked up by zsh, I suppose it just needs to start with _ .